src/service.ts

```diff
@@ -93,12 +94,32 @@ export interface IServiceRequest {
   output: string;
 }
 
+export enum ServiceTypes {
+  rustc,
+  service
+}
+
 export class Service {
-  static async sendRequest(command: string): Promise<IServiceRequest> {
-    const response = await fetch("//wasmexplorer-service.herokuapp.com/service.php", {
+  static async sendRequest(content: string | Object, to: ServiceTypes): Promise<IServiceRequest> {
+    const config = await getConfig();
+    let body;
+    let url;
+    let contentType;
+
+    if (to === ServiceTypes.rustc) {
+      body = JSON.stringify(content);
+      url = config.rustc;
+      contentType = "application/json";
+    } else {
+      body = content.toString();
+      url = config.serviceUrl;
+      contentType = "application/x-www-form-urlencoded";
+    }
```

Comment:
It's hard to the code logic. Could you create two methods `static async sendRequest(content: string | Object, to: ServiceTypes): Promise<IServiceRequest>` and `static async sendRequestJSON(content: string | Object, to: ServiceTypes): Promise<IServiceRequest>` to remove logic around assigning body/url/contentType variables and reduce complexity for simple method?

Reply:
will do